index.html

```diff
@@ -1340,7 +1340,7 @@ <h2 class="no-num no-toc no-ref heading settled" id="subtitle"><span class="cont
      <dt>This version:
      <dd><a class="u-url" href="https://github.com/WICG/cq-usecases">https://github.com/WICG/cq-usecases</a>
      <dt>Issue Tracking:
-     <dd><a href="https://github.com/tomhodgins/cq-usecases/issues/">GitHub</a>
+     <dd><a href="https://github.com/WICG/cq-usecases/issues">GitHub</a>
```

Comment:
This should really changed be in the `.bs` file.

Reply:
https://github.com/WICG/cq-usecases/issues/59

Reply:
@marcoscaceres **index.bs** doesn't contain "Issue Tracking" URL. When the last time **index.html** was generated?

Reply:
Thanks for checking, not sure tbh... @tomhodgins?